easybuild/easyconfigs/j/jax/jax-0.2.18-foss-2020b.eb

```diff
@@ -0,0 +1,90 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Denis Kristak
+easyblock = 'PythonBundle'
+
+name = 'jax'
+version = '0.2.18'
+local_jaxlib_ver = '0.1.70'
+homepage = 'https://pypi.python.org/pypi/jax'
+description = """Composable transformations of Python+NumPy programs: differentiate, vectorize, JIT to GPU/TPU, and
+more"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+# downloading TF tarball for better control of what Bazel will use during installation of Jax
+local_tf_tarball_version = '2.5.0'
+
+builddependencies = [
+    ('Bazel', '3.7.2'),
+]
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('Cython', '0.29.22'),
+    ('opt-einsum', '3.3.0'),
```

Comment:
@deniskristak The easyconfig for `opt-einsum` is missing?

I guess this should be removed, since you have `opt-einsum` included in `exts_list` below...

Reply:
done